picard/util/__init__.py

```diff
@@ -369,6 +375,13 @@ def uniqify(seq):
     return [x for x in seq if x not in seen and not add_seen(x)]
 
 
+def iter_unique(seq):
+    """Creates an iterator only returning unique values from seq"""
+    seen = set()
+    add_seen = seen.add
```

Comment:
I don't get why `add_seen` is needed here.

Reply:
Me neither :D https://github.com/metabrainz/picard/pull/1701/commits/77bf10c0609a95ce838c9ec1795661b1ba1628d3

Reply:
It was added in https://github.com/metabrainz/picard/commit/28b1a0a05b557aaff271771960da51bc09e53921
Not sure why ;)